src/pretalx/agenda/templates/agenda/talk.html

```diff
@@ -150,9 +150,9 @@ <h3 class="talk-title">
                 {% html_signal "pretalx.agenda.signals.html_below_session_pages" sender=request.event request=request submission=submission %}
             </section>
             {% if submission_tags %}
-              {% for tag_item in submission_tags %}
-                  <span class="submission_tag" onclick="#" style="background-color: {{ tag_item.color }}; color: {{ tag_item.contrast_color }};">{{ tag_item.tag }}</span>
-              {% endfor %}
+                {% for tag_item in submission_tags %}
+                    <span class="submission_tag" onclick="#" style="background-color: {{ tag_item.color }}; color: {{ tag_item.contrast_color }};">{{ tag_item.tag }}</span>
+                {% endfor %}
```

Comment:
Same, whitespace changes in the uv feature branch. Please keep as is and make those changes in a separate PR.